.ci/Jenkinsfile

```diff
@@ -392,6 +430,21 @@ def generateStep(Map params = [:]){
   }
 }
 
+def prepareRelease(String nodeVersion='node:lts', Closure body){
+  withNpmrc(secret: "${env.NPMRC_SECRET}", path: "${env.WORKSPACE}/${env.BASE_DIR}") {
+    withCredentials([usernamePassword(credentialsId: '2a9602aa-ab9f-4e52-baf3-b71ca88469c7-UserAndToken',
+                                      passwordVariable: 'GITHUB_TOKEN', usernameVariable: 'GITHUB_USER')]) {
+      docker.image(nodeVersion).inside(){
+        withEnv(["HOME=${env.WORKSPACE}/${env.BASE_DIR}"]) {
+          withTotpVault(secret: "${env.TOTP_SECRET}", code_var_name: 'TOTP_CODE'){
```

Comment:
I don't know if we need to go to the same degree as we did with rum-js here. I assume this is needed for npm in this case as well but I'm not positive.

Reply:
I'm not sure how the npm publish works. In Lerna we were able to use the environment variable `TOTP_CODE` in conjunction with the flag `--otp=`.

What are your thoughts?

Reply:
It doesn't _seem_ like this would be required unless 2FA is required via a configuration flag. I don't see that being the case so far as I can tell, but I would like to confirm with @watson whether or not this is typically a part of the release workflow since the [current release documentation does not make any mention of it](https://github.com/elastic/apm-agent-nodejs/blob/master/CONTRIBUTING.md#releasing).